betterocr/detect.py

```diff
@@ -169,18 +179,16 @@ def detect_boxes(
     items = extract_list(output)
 
     for idx, item in enumerate(items):
-        box = item["box"]
-
         # [x,y,w,h]
-        if len(box) == 4 and isinstance(box[0], int):
-            rect = rectangle_corners(box)
-            items[idx]["box"] = rect
+        if len(item) == 4 and isinstance(item[0], int):
+            rect = rectangle_corners(item["box"])
+            items[idx] = rect
 
         # [[x,y],[w,h]]
-        elif len(box) == 2 and isinstance(box[0], list) and len(box[0]) == 2:
-            flattened = [i for sublist in box for i in sublist]
+        elif len(item) == 2 and isinstance(item[0], list) and len(item[0]) == 2:
+            flattened = [i for sublist in item for i in sublist]
             rect = rectangle_corners(flattened)
-            items[idx]["box"] = rect
+            items[idx] = rect
```

Comment:
Can you rebase your commit? This line had also been changed

Reply:
Rebased